content/when-should-we-automate-tests.md

```diff
@@ -0,0 +1,34 @@
+Title: When should we automate tests
+Date: 2022-01-01
```

Comment:
@wsbctechnicalblog you will have to update this date B4 publishing.

Reply:
@wsbctechnicalblog I have fixed the image issue for @adichourasiya and added two empty lines at the end. The post is good to go and will be our first QA post - I assume that @adichourasiya will add the promised video link in future.